zzz_allTables.Rmd

```diff
@@ -0,0 +1,79 @@
+---
+title: "**All Tables**"
```

Comment:
Does this file still need to be called `zzz_`?

Reply:
No. I changed it's name to allTables.Rmd. I think it is now showing outdated on github.